### PR TITLE
Taken Courses as a separate screen

### PR DESCRIPTION
**Pull request by failita**
- addedTaken Courses as a separate screen
- replaced button color with CieColor
- added some static variables to the settings page
- changed some tests due to the static variables and text changes

### CODE REVIEW

**Comment by AndiAndreas** [comment]
I thought that we wanted to make a scrollable settings screen. We although talked about that last week in class.
See: #38

**Comment by failita** [comment]
Oh, ok. I must have misunderstood.

**Comment by failita** [comment]
I think a separate screen might be good if we want to add a delete function for taken courses.

File: lib/views/login.dart [hunk 1]
```diff
@@ -138,7 +138,7 @@ class LoginFormState extends State<LoginForm> {
                   new Container(
                     padding: const EdgeInsets.fromLTRB(25.0, 10.0, 25.0, 10.0),
                   child: new FlatButton(
-                    color: CiEStyle.getLogoutButtonColor(),
```
Comment (AndiAndreas) [inline lib/views/login.dart]: Could you please use the CiEStyle here?

File: lib/views/start.dart [hunk 2]
```diff
@@ -55,7 +55,7 @@ class _WelcomePageState extends State<WelcomePage> with SingleTickerProviderStat
           new Container(
             padding: const EdgeInsets.all(25.0),
             child: new FlatButton(
-              color: CiEStyle.getLogoutButtonColor(),
+              color: CiEColor.red,
```
Comment (AndiAndreas) [inline lib/views/start.dart]: Please use CiEStyle Colors here

**Comment by AndiAndreas** [comment]
Looks good as well! NICE

File: lib/views/login.dart (hunk 1 above)
Comment (failita) [reply lib/views/login.dart]: sure

**Comment by RedTo** [comment]
Could this branch be deleted?

**Comment by failita** [comment]
Yes, I'll delete it shortly.

**Pull request merged**